packages/react/src/combobox/ComboboxBase/ComboboxListItem.tsx

```diff
@@ -1,52 +1,79 @@
+import { type CSSObject } from '@emotion/react';
 import { forwardRef, HTMLAttributes } from 'react';
-import { boxPalette, mapSpacing, packs, tokens } from '../../core';
+import { boxPalette, fontGrid, mapSpacing, packs, tokens } from '../../core';
 import { useIsIos } from '../utils';
 
-type ComboboxListItemProps = Omit<HTMLAttributes<HTMLLIElement>, 'color'> & {
-	isActiveItem: boolean;
-	isInteractive: boolean;
-};
+type ComboboxListItemProps = Omit<HTMLAttributes<HTMLLIElement>, 'color'>;
 
 const hoverStyles = {
 	color: boxPalette.foregroundText,
 	backgroundColor: boxPalette.backgroundShade,
 	'> span': packs.underline,
 };
 
+const FONT_SIZE_LINE_HEIGHT = fontGrid('sm', 'default');
+
+export const listItemStyles: CSSObject = {
+	'[data-combobox-list-item]': {
```

Comment:
Migrated everything to data attributes and this css object is used one in the base component instead of every list item